org.texmacs.TeXmacs.yml

```diff
@@ -0,0 +1,116 @@
+command: texmacs
+rename-desktop-file: texmacs.desktop
+rename-icon: texmacs
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --persist=.TeXmacs
+  - --env=PATH=/app/texlive/bin/:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin
+  - --env=ASPELL_CONF=dict-dir /app/share/dicts
+  - --env=LD_LIBRARY_PATH=/app/texlive/lib:/app/texlive/lib/perl5/5.38.2/aarch64-linux/CORE:/app/texlive/lib/perl5/5.38.2/x86_64-linux/CORE
+  - --env=PERL5LIB=/app/texlive/lib/perl5/5.38.2:/app/texlive/lib/perl5/site_perl/5.38.2
+
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    version: "23.08"
+    directory: texlive
+    autodelete: false
+  org.lyx.LyX.Plugin:
+    versions: stable;beta;test
+    directory: plugins
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+    add-ld-path: lib
+modules:
+  - name: texliveYextensions
+    buildsystem: simple
+    build-commands:
+      - install -d /app/texlive
+      - install -d /app/plugins
+  - name: magick
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/ImageMagick/ImageMagick/archive/refs/tags/7.1.1-21.tar.gz
+        sha256: 09402e5f17c6575ef9f010bb2e21ae1710f1f3426f115ad4317ee9129c32608e
+  - name: graphviz
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://gitlab.com/api/v4/projects/4207231/packages/generic/graphviz-releases/10.0.1/graphviz-10.0.1.tar.gz
+        sha256: 72e1a575e9c382c4fd2267837962bc4582b3d9ad746e338cd8024bdeca124f32
+  - name: gnuplot
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/gnuplot/5.4.10/gnuplot-5.4.10.tar.gz
+        sha256: 975d8c1cc2c41c7cedc4e323aff035d977feb9a97f0296dd2a8a66d197a5b27c
+  - name: guile
+    buildsystem: autotools
+    config-opts:
+      - --disable-static 
+      - --disable-error-on-warning
+    sources:
+      - type: archive
+        url: http://ftp.gnu.org/gnu/guile/guile-1.8.8.tar.gz
+        sha256: c3471fed2e72e5b04ad133bbaaf16369e8360283679bcf19800bc1b381024050
+      - type: patch
+        path: guile-1.8.8-configure.patch
+  - name: freetype
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://download.savannah.gnu.org/releases/freetype/freetype-2.13.2.tar.xz
+        sha256: 12991c4e55c506dd7f9b765933e62fd2be2e06d421505d7950a132e4f1bb484d
+
+  - name: dictEs-aspell
+    buildsystem: simple
+    build-commands:
+      - ./configure
+      - make datadir=/app/share/dicts dictdir=/app/share/dicts
+      - make datadir=/app/share/dicts dictdir=/app/share/dicts install
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/aspell/dict/es/aspell6-es-1.11-2.tar.bz2
+        sha256: ad367fa1e7069c72eb7ae37e4d39c30a44d32a6aa73cedccbd0d06a69018afcc
+  - name: dictEn-aspell
+    buildsystem: simple
+    build-commands:
+      - ./configure
```

Comment:
should this be `buildsystem: autotools` ?

Reply:
the --prefix gives an error.